components/datetime/tests/fixtures/tests/styles.json

```diff
@@ -101,7 +101,7 @@
             }
         },
         "output": {
-            "value": "sobota, 21 marca 2020 08:25:07 zzzz"
+	    "value": "субота, 21 сакавіка 2020 'г'. 'у' 08:25:07, zzzz"
```

Comment:
Looks like a small bug; I think you are supposed to remove the quotes.  Can you file a follow-up issue?

Reply:
https://github.com/unicode-org/icu4x/issues/319